.github/workflows/patch-code-of-conduct.yml

```diff
@@ -0,0 +1,26 @@
+name: Patch CoC
+on: [push]
```

Comment:
this should also be on a cron

Reply:
I just added the cronjob. Thank you for the review.